TestData/SelfVerifyingTests/VBToCS/ExpressionTests.vb

```diff
@@ -7,30 +7,37 @@ Module Program
     Public Class Tests
 
         <Fact>
-        Public Sub TestFloatingPointDivision()
+        Public Sub TestFloatingPointDivisionOfIntegers()
             Dim x = 7 / 2
             Assert.Equal(x, 3.5)
         End Sub
 
         <Fact>
-        Public Sub TestIntegerDivision()
+        Public Sub TestIntegerDivisionOfIntegers()
             Dim x = 7 \ 2
             Assert.Equal(x, 3)
         End Sub
 
+        <Fact>
+        Public Sub TestDecimalDivisionOfDecimals()
+            Dim x = 7D / 2D
+            Assert.Equal(x, 3.5D)
+        End Sub
+        
+        <Fact>
+        Public Sub TestIntegerDivisionOfChars()
+            Dim x As Char = 7
+            Dim y As Char = 2
+            Dim z = x / y
+            Assert.Equal(x, 3)
+        End Sub
```

Comment:
This turned out not to be a thing - my Visual Studio was playing up so I couldn't try it out